rfcs/0000-data-structures/0000-data-structures.md

```diff
@@ -0,0 +1,265 @@
+---
+Number: "0000"
+Category: Standards Track
+Status: Proposal
+Author: Haichao Zhu, Xuejie Xiao
+Organization: Nervos Foundation
+Created: 2019-03-26
+---
+
+# Data Structures of Nervos CKB
+
+This documents list all the basic structures one may need to know in order to develop on CKB, or mine CKB coins.
```

Comment:
Shall we delete this sentence?
I think it's not suitable for an RFC document anymore.

Reply:
I think it serves as a summary of the doc, what do you think?

Reply:
I've slightly revised this a bit, let me know what you think.

Reply:
lgtm